### DISCUSSION TITLE
alivecc: emitting only interested passes validation results

### DISCUSSION THREAD

**Discussion by BalaRishi-AMD** [AliveToolkit/alive2#737] [Q&A] [answered]
Hi,

How to restrict alivecc to emit validation results only for selected passes (passes of my interested) ?

Thanks.

**Comment by regehr** [comment, marked as answer]
afaik there's not a builtin mechanism for this, but it would be easy to implement, see the do_skip() function in tv.cpp. it should be trivial to modify this function to skip all passes that you're not interested in.
